core/dbt/rpc/task_handler.py

```diff
@@ -202,11 +203,12 @@ def handle_completed(self):
             if self.handler.result is None:
                 # there wasn't an error before, but there sure is one now
                 self.handler.error = dbt_error(
-                    dbt.exceptions.InternalException(
+                    InternalException(
                         'got an invalid result=None, but state was {}'
                         .format(self.handler.state)
                     )
                 )
+            # TODO: need to tighten RequestTaskHandler.Task to also
```

Comment:
Is there more to this comment?

Reply:
I think there's supposed to be less to that comment - removing it!